enhancements/update/channel-rename-generally-available.md

```diff
@@ -0,0 +1,354 @@
+1. The cluster admin acquires a 4.16.z installer binary.
+  * Signed installer binaries are available on the mirrors, e.g. [amd64-fast-4.14][mirror-fast-4.14].
+  * [Docs][install-docs] point at [console.redhat.com][console-create] which in turn [points][console-install-user-provisioned] into the mirrors with `stable` links like [this][mirror-stable-installer].
+    This path does not currently seem to point out the existence o [the sibling GPG signature][mirror-stable-signature].
+  * Installer binaries (with hashes) are also linked from [the OCP portal page][portal-ocp], with URIs that include the version and hashes, but which do not include any channel names.
+1. The cluster admin runs the installer, which generates a ClusterVersion manifest declaring [a default channel that is manually maintained][installer-default-channel].
+   New for this enhancement, that default channel will become `fleet-approved-4.y`.
+
+#### Variation and form factor considerations
+
+HyperShift takes [`spec.channel`][HostedCluster-spec-channel] and the [`desired` Release][HostedCluster-status-version-desired] all the way up to the HostedCluster level, so the channel renames should be transparent there.
+
+Microshift manages installs via RPMs, without using an update service, so channels are not relevant there.
+
+Managed OSD/ROSA collapse channels down into channel-groups, so we'll need to work with them on that.
+
+Possibly the assisted-installer folks have tooling that uses channels to track installable versions, so we'll want to round with them about that.
+
+### API Extensions
+
+No API extensions needed.
+
+### Implementation Details/Notes/Constraints
+
+#### Historical context
+
+History of OpenShift channel handling:
+
+* OpenShift 4.5 and earlier, the in-cluster web console [manually maintained a list of expected channels][web-console-4.15-channels].
+* OpenShift 4.6:
+  * The cluster-version operator [started setting `status.desired.channels`][cluster-version-status-channels] ([enhancement](available-update-metadata.md)).
+  * The in-cluster web console [started passing `status.desired.channels` on][web-console-consumes-channels] to cluster admins selecting their cluster's channel.
+* OpenShift 4.8, the in-cluster web console [dropped the hard-coded fallback channels][web-console-drops-fallback-channels].
+* OpenShift 4.9, `oc adm upgrade` [started passing `status.desired.channels` on][oc-adm-upgrade-channel] to cluster admins, and added a `channel` subcommand to edit the channel.
+
+Since then, channels are largely a negotiation between cluster admins selecting a channel and the update service declaring a set of channel choices, with OCP-level components acting as intermediates passing around opaque channel names.
+
+#### graph-data
+
+The graph-data repository will require several modifications, for example updating [channel-creating scripts][graph-data-release-ga] to use the new names.
+
+#### Docs exposure
```

Comment:
Docs in the managed openshift context also need updated.

Impacts around ARO statements for supporting stable only.
https://learn.microsoft.com/en-us/azure/openshift/openshift-service-definitions

ROSA (and OSD) reference to "stable" Channel Group.
https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-creating-cluster.html

OCM and ROSA CLI docs and probably KCS, especially where we have called out how to use non-stable for some external organizations.

And probably more..

Reply:
I've added some references to these in the docs section, but I've tried to have those link out to the managed/ARO sections, because "docs or shipped code?" seems like a less-core distinction than "managed or OCP-core?".